manifests/eip.pp

```diff
@@ -15,15 +15,12 @@
         $file_enure         = 'absent'
     }
 
-    if $auto == true {
-        $link_ensure        = 'link'
-    } else {
-        $link_ensure        = 'absent'
-    }
-
-    file { "/etc/nubis.d/${order}-eip-associate":
-        ensure  => $link_ensure,
-        target  => '/usr/local/sbin/eip-associate',
+    if $auto {
+        file { "/etc/nubis.d/${order}-eip-associate":
+            ensure  => $link_ensure,
```

Comment:
$link_ensure no longer exists in the template. Is this inherited from elsewhere or is this an over-site?